categorization-svm-2/Merge product and product nuts.ipynb

```diff
@@ -43,10 +43,13 @@
    },
    "outputs": [],
    "source": [
-    "products = '/Users/elise/Documents/?/? data/new/products.txt'\n",
-    "products_nuts = '/Users/elise/Documents/?/? data/new/product_nuts.txt'\n",
+    "#products = '/Users/elise/Documents/?/? data/new/products.txt'\n",
+    "#products_nuts = '/Users/elise/Documents/?/? data/new/product_nuts.txt'\n",
+    "products = '/data/products.txt'\n",
+    "products_nuts = '/data/product_nuts.txt'\n",
```

Comment:
This will most probably not work, as you're using an absolute path (starting with `/`). Replace this with `products = 'data/products.txt'`.